docs/relying-parties/reference/sub-plat-contentful.md

```diff
@@ -23,201 +16,85 @@ You can log into Contentful through [Mozilla Dashboard](https://sso.mozilla.com/
+| Common content   | Optional. Content used on multiple pages (e.g., Privacy Notice URL, TOS URL, Success Action Button URL, etc.), not specific to a certain component. If the product offering is available in other locales, translation fields appear in this entry, once locales have been selected.<br/>- Product managers are to create and add this entry to Offering, if applicable.                                                                                                                     |
+| Coupon Config    | Optional. Coupon configuration options of the Stripe Promotion Codes valid in specified countries for the product offering. <br/>- Product managers are to create and add this entry to Offering, if applicable.                                                                                                                                                                                                                                                                             |
+| IAP              | Optional. IAP configurations that include the Apple App Store productIds and/or Google Play product SKUs (now called product IDs) that are mapped to this Offering. <br/>- Product managers are to create and add this entry to Offering, if applicable.                                                                                                                                                                                                                                     |
+| Offering         | **Required.** The configuration of the subscription product. This entry includes the Stripe Product ID from Stripe of the product offering, and requires the Capabilities and Purchase entries. If applicable, this entry should be added to a Subgroup if a customer is able to upgrade from or to this product offering (see Subgroup). <br/>- Product managers are to create this entry.                                                                                                  |
+| Purchase         | **Required.** This entry requires the related Purchase Details and Offering entries to the product offering. If applicable, it should include all active Stripe Plan IDs from Stripe associated to the Stripe Product ID of the product offering.<br/>- Product managers are to create and add this entry to Offering.                                                                                                                                                                       |
```

Comment:
Not all active Stripe Plan ID's, only the new multi-currency ones that should be available on the Checkout page for this product in SubPlat 3. Legacy single-currency product plans go on the Offering Stripe Legacy Plans field.